frontend/src/app/modules/forgot-password/write-email/write-email.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { AuthService } from '@core/services/auth.service';
+
+@Component({
+    selector: 'app-write-email',
+    templateUrl: './write-email.component.html',
+    styleUrls: ['./write-email.component.sass'],
+})
+export class WriteEmailComponent {
+    emailPattern: RegExp = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
```

Comment:
The same with regex

Reply:
i haven't seen regex for mail in other components